### ISSUE TITLE
How to exclude .DS_Store files

### ISSUE THREAD

**Issue by Drizzt2028** [qweasdd/manga-colorization-v2#10]
When running on macOSX, an error will be reported because there is a .DS_Store file in the directory.

**Comment by Drizzt2028** [comment]
def colorize_images(target_path, colorizator, args):
    images = os.listdir(args.path)
    for files in os.listdir(args.path):
        for filename in files:
            if filename == '.DS_Store':
                    args_path = os.path.join(root, filename)
                    os.remove(args_path)
            continue

**Comment by Drizzt2028** [comment]
` name, ext = os.path.splitext(image_name)
        if (image_name == '.DS_Store'):
            os.remove(args.path,'.DS_Store')
            
        print("Delete .DS_Store OK")` Worked

**Comment by Keiser04** [comment]
in the inference.py? i'm editing a fork to make it more visual (gradio) so i can fix it for mac users too.

**Comment by arthurwolf** [comment]
@Keiser04 got a link for the fork?

**Comment by Keiser04** [comment]
https://github.com/Keiser04/manga-colorization-v2

copy me the error and send it to me either here or in an issue in my fork

so you save me work and I try to fix the error.

**Comment by DDXDB** [comment]
@Keiser04 Do you know how to train

**Comment by Keiser04** [comment]
@DDXDB Yes but the model only works in v1, the v2 has encoders but idk how i can do

**Comment by Keiser04** [comment]
but... idk how to use. this is my best result
![image](https://github.com/qweasdd/manga-colorization-v2/assets/146493916/4bc3a8d5-4fc0-4b33-bd4c-f741dd582706)

**Comment by arthurwolf** [comment]
is that result with the default provided model, or one you trained yourself ?
anyone around here interrested on sharing efferts on training (like working on preparing a dataset together, and on training together ?)

**Comment by Keiser04** [comment]
It is my model

**Comment by Keiser04** [comment]
> is that result with the default provided model, or one you trained yourself ? anyone around here interrested on sharing efferts on training (like working on preparing a dataset together, and on training together ?)

Look inside my repository, I have made some to facilitate the dataset. if you want I can share it with you but the results are not the desired and it takes a long time.

**Comment by arthurwolf** [comment]
> Look inside my repository, I have made some to facilitate the dataset. if you want I can share it with you but the results are not the desired and it takes a long time.

I plan on starting a dataset of my own in a few weeks. Would you be interested in joining efforts? Put our datasets together (and maybe find other participants here too?), and also share efforts on training. The result would be a model better than what either of us can do. If you're ready to share your dataset/model, I'll come back to you when I have my own and merge the two, and see what training with the merged dataset does.

As they say, Unity is Strength.

**Comment by Keiser04** [comment]
> > Look inside my repository, I have made some to facilitate the dataset. if you want I can share it with you but the results are not the desired and it takes a long time.
> 
> I plan on starting a dataset of my own in a few weeks. Would you be interested in joining efforts? Put our datasets together (and maybe find other participants here too?), and also share efforts on training. The result would be a model better than what either of us can do. If you're ready to share your dataset/model, I'll come back to you when I have my own and merge the two, and see what training with the merged dataset does.
> 
> As they say, Unity is Strength.

Let's see, yes, I can share my dataset with you, but I honestly don't trust the results I had.

**Comment by Mlegbder** [comment]
def colorize_images(target_path, colorizator, args):
    images = os.listdir(args.path)
    
    for image_name in images:
        file_path = os.path.join(args.path, image_name)
        
        if os.path.isdir(file_path):
            continue
        
        name, ext = os.path.splitext(image_name)
        if (image_name == '.DS_Store'):
            continue
        if (ext != '.png'):
            image_name = name + '.png'
        
        print(file_path)
        
        save_path = os.path.join(target_path, image_name)
        colorize_single_image(file_path, save_path, colorizator, args)

**Comment by Mlegbder** [comment]
inference.py

`def colorize_images(target_path, colorizator, args):
    images = os.listdir(args.path)
    
    for image_name in images:
        file_path = os.path.join(args.path, image_name)
        
        if os.path.isdir(file_path):
            continue
        
        name, ext = os.path.splitext(image_name)
        if (image_name == '.DS_Store'):
            continue
        if (ext != '.png'):
            image_name = name + '.png'
        
        print(file_path)
        
        save_path = os.path.join(target_path, image_name)
        colorize_single_image(file_path, save_path, colorizator, args)`
